src/FSharp.Control.TaskSeq/Utils.fsi

```diff
@@ -80,25 +83,22 @@ module Task =
     /// Bind a Task<'T>
     val inline bind: binder: ('T -> #Task<'U>) -> task: Task<'T> -> Task<'U>
 
-    /// Create a task from a value
-    val inline fromResult: value: 'U -> Task<'U>
-
 module Async =
 
     /// Convert an Task<'T> into an Async<'T>
     val inline ofTask: task: Task<'T> -> Async<'T>
 
-    /// Convert a unit-task into an Async<unit>
+    /// Convert a non-generic Task into an Async<unit>
     val inline ofUnitTask: task: Task -> Async<unit>
 
-    /// Convert a Task<'T> into an Async<'T>
+    /// Starts a running instance of an Async<'T>, represented as a Task<'T>
```

Comment:
Actually, this is incorrect. The `Async` does not have to be running. Tasks are hot-started, asyncs are not. It is more likely it is not running. Maybe this?
```suggestion
    /// Immediately starts an Async<'T> and returns it as a Task<'T>
```

Reply:
>The Async does not have to be running

If you're referring to things like an Async wrapping a Task or a StartChild, I'm more thinking about the fact that this in the general case will start work

> <suggestion>

Probably. I think of Async as a Builder, and any of the StartX things as doing a Create (or a Task.Run).
I'm trying to convey unambiguously to people that might have not have the hot vs cold thing sorted out and/or are not paying attention that this kicks off a new thing, an that continuations will be in the threadpool
The fact that the input is an Async<'T> is immaterial

Trying to cover both angles, starting from:

> `/// Immediately starts an Async<'T> and returns it as a Task<'T>`

I would say (if wanted to convey that it kicks off new work, and that the new work is a Task

`/// Starts the computation associated with the `Async<'T>` inline, returning the associated `Task<'T>`

But I don't like that; Maybe:

`/// Runs the `Async<'T>` computation via `StartImmediateAsTask`, returning the associated `Task<'T>`

**The main thing I'm trying to convey to someone sprikling `Async.toTask` around is that in the general case new parallel work is kicking off in the threadpool when you do this, even if technically we could be mapping a wrapped handle from a StartChild or AwaitTask**

Reply:
> that this in the general case will start work

What I meant was "it starts a running car". It is a double entendre. You either "start a car (and make it run)" or "continue running (if it already ran, but didn't stop yet)".

Same with asyncs, right? The typical `Async<'T>` instance is not running, until you call `StartAsTask` or something.

Reply:
Btw, under the hood, it appears to call `System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start`, which may or may not be the same as `StartImmediateAsTask`...

Reply:
Ah, yes that does make sense (your objection to "starts a running" et). I will try to distil all of the above down to a change and then we go again...

Reply:
Went with 

> Starts the `Async<'T>` computation, returning the associated `Task<'T>`

for both. I wanted a 'starts' or a 'running' or 'instance' in the wording to convey that in 90% of cases it kicks off work and is not just converting something in some very boring mechanical manner like the typical `of`/to` pairs

And I definitely agree that mentioning or lying about using StartImmediateAsTask is not going to help people regardless of whether they even know that that even is?

There's still. the sister comment thread open; Mark this Resolved if you can live with the wording and/or can't thnk of a better one right now

Reply:
> in 90% of cases it kicks off work and is not just converting something in some very boring mechanical manner like the typical of/to` pairs

Indeed. To be fair, if you convert something to a `task` (as in, the F# version of a hot task), it implies that you immediately start it. I don't think there is a way to convert it to a `task` that is not hot-started, and keep the same F#-task semantics. 

In fact, it was the whole point of `task {...}` to begin with: a hot-started computation expression for async stuff.

>  Mark this Resolved if you can live with the wording and/or can't thnk of a better one right now

👍 totally good! I like it.

Reply:
Good catch; I simply dropped the ball; should be fixed now (and I rebased 2m ago)